tensorboard/plugins/graph/keras_util.py

```diff
@@ -255,7 +255,19 @@ def keras_model_to_graph_def(keras_layer):
                     inbound_node_names = model_name_to_output.get(
                         inbound_name, [inbound_name]
                     )
-                    node_def.input.append(inbound_node_names[index])
+                    # There can be multiple inbound_nodes that reference the
+                    # same upstream layer. This causes issues when looking for
+                    # a particular index in that layer, since the indices
+                    # captured in `inbound_nodes` doesn't necessarily match the
+                    # number of entries in the `inbound_node_names` list. To
+                    # avoid IndexErrors, we just use the last element in the
```

Comment:
Can we clarify something along the lines of "this is a quick hack, that might or might not be an appropriate solution to this problem."? As it is, it might read as if "we've thought about this, and this is a reasonable solution", but really, we don't know that it is.

Reply:
Good point, left a comment to the same effect in the upstream bug but didn't note it here. Updated!